readme.md

```diff
@@ -6,6 +6,11 @@ eBPF is a pure Go library that provides utilities for loading, compiling, and de
 
 [ebpf/asm](https://godoc.org/github.com/cilium/ebpf/asm) contains a basic assembler.
 
+## Current status
+
+The package is production ready, but the API is explicitly unstable
```

Comment:
I would put `the API is explicitly unstable right now` in bold;-)

Reply:
maybe just "right now" in bold?